### ISSUE TITLE
Changing default workflow doesn't work

### ISSUE THREAD

**Issue by drphero** [ai-dock/comfyui#127]
When using the Flux provisioning script and changing the default workflow variable, this doesn't actually affect what the default workflow is.

**Comment by robballantyne** [comment]
@drphero yeah this broke in a recent ComfyUI update. I'm going to check for an alternative method of setting the default if possible
